src/humanize/time.py

```diff
@@ -396,7 +413,12 @@ def _suppress_lower_units(min_unit, suppress):
     return suppress
 
 
-def precisedelta(value, minimum_unit="seconds", suppress=(), format="%0.2f") -> str:
+def precisedelta(
+    value: dt.timedelta | int,
+    minimum_unit: str = "seconds",
+    suppress: typing.Iterable[str] = (),
```

Comment:
Use `collections.abc.Iterable` instead. Using it from `typing` has been [deprecated](https://docs.python.org/3/library/typing.html#typing.Iterable) since 3.9, although we've got 5 years to make the change.

Reply:
Good to know, will replace.

Reply:
Or use `typing.Sequence`? Not sure this would fit here, just my 2 cents 🙂